### PR TITLE
fix: remove kratos and version check from boot

### PR DESCRIPTION
**Pull request by rebelchris**
Since we reached a maturity on kratos users, we no longer need to support the old checks.

WT-937 #done

### CODE REVIEW

File: src/routes/boot.js [hunk 1]
```diff
@@ -80,84 +74,65 @@ export const bootSharedLogic = async (ctx, shouldRefreshToken) => {
...
-            user: {
-              ...base.user,
-              ...user,
-              providers: [userProvider?.provider],
-              roles,
-              permalink: `${config.webappOrigin}/${user.username || user.id}`,
-            },
-            accessToken,
+            ...returnObject,
+            registrationLink: `${config.webappOrigin}/register`,
```
Comment (idoshamun) [inline src/routes/boot.js:119]: Do we still use it?

**Review by idoshamun — APPROVED** [review]
Thanks a lot of cleaning it up

File: src/routes/boot.js (hunk 1 above)
Comment (rebelchris) [reply src/routes/boot.js:119]: Ah no good point, we can also remove this :)

**Review by sshanzel — APPROVED** [review]
Thanks for cleaning the legacy code! 🚢 
Nothing looks weird to me, nice work! 💪

**Pull request merged**